src/test/resources/dist-files/installer-version

```diff
@@ -0,0 +1 @@
+8a169b9b-ff04-44cd-8ad9-6761ab6fd9b5
```

Comment:
@suganyasuven this is only required for testings right? shall we move into test resources and keep ballerina-version in original location?